src/Invoker.php

```diff
@@ -53,6 +61,19 @@ public function handle(
             $request = ServerRequest::fromGlobals();
         }
 
-        return $this->function->execute($request);
+        try {
+            return $this->function->execute($request);
+        } catch (\Exception $e) {
+            // Log the full error and stack trace
+            ($this->errorLogFunc)((string) $e);
+            // Set "X-Google-Status" to "crash" for Http functions and "error"
+            // for Cloud Events
+            $statusHeader = $this->function instanceof HttpFunctionWrapper
+                ? 'crash'
+                : 'error';
+            return new Response(200, [
+                FunctionWrapper::FUNCTION_STATUS_HEADER => $statusHeader,
+            ], $e->getMessage());
```

Comment:
This error message might surface implementation details of the function, maybe it should respect [display_errors](https://www.php.net/manual/en/errorfunc.configuration.php#ini.display-errors)?

Reply:
Yes, I am also okay omitting it to be safe

Reply:
I'll remove the error from the response for simplicity for now